### PR TITLE
Add contributor guidelines file and start discussion around dao-kit contribution

### PR DESCRIPTION
**Pull request by benstew**
Meets requirements for #12 with boilerplate contribution file.

Also, I wanted to spark up a discussion around contribution to the dao-kits monorepo based on my initial experience. I know there are a lot of advancements slated for this repo (eg TCR, Payroll, and the BareKit) but I had two primary pain points as an external contributor that I wanted to mention. 

The good news is that I'm interested in lending a hand to solve those issues for future contributors :)

**Pain Point 1: Limited documentation**
There is great power in kits and templates but I found it challenging to understand the future direction which makes thinking through documentation (like contribution guidelines) a little challenging. With upcoming improvements, specifically BareKit, I would love to understand how we envision the evolution of kits. 

Honestly, I think this could be as easy as updated the README and maybe a short blog post or wiki entry. I think this would allow potential contributors to really conceptualize the power, flexibility, and importance of kits/templates in the system.

**Pain Point 2: Kit code security & audit processes**
Modification to kit code will be scrutinized (potentially more than other areas) and would love to learn more about the form this will take in the near term. It's mentioned that once available, all mainnet-ready kits will leverage a `security.md` file that essentially flags potential issues/concerns that deviate from a best practice approach. I see the value in adding a lightweight version of that for all kits. Would be great to also get some form minimal integration test (even now, if not done already)

To sum up, I'm interested in learning more about the future development of dao-kits and determining what quick, high value add improvements could be done to make this repo more friendly to outside contributors.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Contributing to Aragon DAO Kits and Templates
```
Comment (benstew) [inline CONTRIBUTING.md]: Boilerplate language that I'm happy to update based on the discussion. Wanted to get a better picture of the future before trying to change too much.

Comment (izqui) [reply CONTRIBUTING.md]: I think this looks good

**Comment by benstew** [comment]
Thanks for the review @izqui. 

I can open up a new issue for updating the README since I won't get to that this week. And great to know about the integration tests. Now I can sleep at night ;)

Also, I will probably focus on the TCR work that the team has been focused. I have a keen interest in TCRs and I think that's where I will look to add value next.

**Comment by izqui** [comment]
Awesome, thanks to you!
